Adds a PR template

cdrage opened the pull request:
**What kind of PR is this?**
<!--
For example:
- Bug fix
- Enhancement
- Feature
-->

Documentation / GitHub

**What does does this PR do / why we need it**:

This adds a PR template whenver someone opens a new PR.

**Which issue(s) this PR fixes**:

N/A

**How to test changes / Special notes to the reviewer**:

N/A

<!---
Example:
```sh
helm install
your steps here
```
-->

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

bacongobbler commented:
can you use the pr template in the Helm 2 branch? https://github.com/helm/helm/blob/dev-v2/.github/PULL_REQUEST_TEMPLATE.md

cdrage committed:
Adds a PR template

**What kind of PR is this?**
<!--
For example:
- Bug fix
- Enhancement
- Feature
-->

Documentation / GitHub

**What does does this PR do / why we need it**:

This adds a PR template whenver someone opens a new PR.

**Which issue(s) this PR fixes**:

N/A

**How to test changes / Special notes to the reviewer**:

N/A

<!---
Example:
```sh
helm install
your steps here
```
-->

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

cdrage commented:
@bacongobbler Sure! I've updated the PR to include some of the comments that were added to https://github.com/helm/helm/blob/dev-v2/.github/PULL_REQUEST_TEMPLATE.md
